docs/markdown/Reference-manual.md

```diff
@@ -1664,9 +1664,10 @@ test(..., env: nomalloc, ...)
 
 - `protocol` specifies how the test results are parsed and can be one
   of `exitcode` (the executable's exit code is used by the test harness
-  to record the outcome of the test) or `tap` ([Test Anything
-  Protocol](https://www.testanything.org/)). For more on the Meson test
-  harness protocol read [Unit Tests](Unit-tests.md). Since 0.50.0
+  to record the outcome of the test), `tap` ([Test Anything
+  Protocol](https://www.testanything.org/)) or `gtest`. For more on the Meson
+  test harness protocol read [Unit Tests](Unit-tests.md). Since 0.50.0.
+  *(Since 0.55.0)* The `gtest` protocol.
```

Comment:
This might be best written as a bulleted list.

Reply:
I agree, I've updated it to be a list